Neos.Kickstarter/Classes/Service/GeneratorService.php

```diff
@@ -386,6 +387,45 @@ public function generateDocumentation($packageKey)
     }
 
     /**
+     * Generate translation for the package key
+     *
+     * @param string $packageKey
+     * @param string $sourceLanguageKey
+     * @param array $targetLanguageKeys
+     * @return array An array of generated filenames
+     */
+    public function generateTranslation($packageKey, $sourceLanguageKey, array $targetLanguageKeys = [])
+    {
+        $translationPath = Files::concatenatePaths([$this->packageManager->getPackage($packageKey)->getResourcesPath(), 'Private/Translations']);
+        $contextVariables = [];
+        $contextVariables['packageKey'] = $packageKey;
+        $contextVariables['sourceLanguageKey'] = $sourceLanguageKey;
+
+        $templatePathAndFilename = 'resource://Neos.Kickstarter/Private/Generator/Translations/SourceLanguageTemplate.xlf.tmpl';
+        $fileContent = $this->renderTemplate($templatePathAndFilename, $contextVariables);
+        $targetPathAndFilename = Files::concatenatePaths([$translationPath, $sourceLanguageKey, 'Main.xlf']);
+        $this->generateFile($targetPathAndFilename, $fileContent);
+
+        if ($targetLanguageKeys) {
+            $sourceLanguageFile = Files::concatenatePaths([$this->packageManager->getPackage($packageKey)->getResourcesPath(), 'Private/Translations', $sourceLanguageKey, 'Main.xlf']);
+
+            $xliffParser = new XliffParser();
+            $parsedXliffArray = $xliffParser->getParsedData($sourceLanguageFile);
+            foreach ($targetLanguageKeys as $targetLanguageKey) {
+                $contextVariables['targetLanguageKey'] = $targetLanguageKey;
+                $contextVariables['translationUnits'] = $parsedXliffArray['translationUnits'];
+
+                $templatePathAndFilename = 'resource://Neos.Kickstarter/Private/Generator/Translations/TargetLanguageTemplate.xlf.tmpl';
+                $fileContent = $this->renderTemplate($templatePathAndFilename, $contextVariables);
+                $targetPathAndFilename = Files::concatenatePaths([$translationPath, $targetLanguageKey, 'Main.xlf']);
+                $this->generateFile($targetPathAndFilename, $fileContent);
```

Comment:
Currently the output looks like this:
![image](https://cloud.githubusercontent.com/assets/5312377/24458560/8ef6417a-1499-11e7-9156-5ba940769c1e.png)

I would customize the output of `generateFile` to `Omitted as file already exists ...`, as it effects all kickstarters in the same way. What do you think?